desktop/core/src/desktop/auth/views.py

```diff
@@ -145,7 +145,7 @@ def dt_login(request, from_modal=False):
         userprofile = get_profile(user)
 
         login(request, user)
-
+        # If Test cookie exists , it should be deleted
```

Comment:
Really needed comment?

Reply:
Was added for self understanding, thought it will help others for readability.
if required will remove it :)